pkg/kudoctl/cmd/init.go

```diff
@@ -102,6 +103,7 @@ func newInitCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 	f := cmd.Flags()
 	f.BoolVarP(&i.clientOnly, "client-only", "c", false, "If set does not install KUDO on the server")
 	f.StringVarP(&i.image, "kudo-image", "i", "", "Override KUDO controller image and/or version")
+	f.StringVarP(&i.imagePullPolicy, "pull-policy", "", "", "Override KUDO controller image pull policy")
```

Comment:
```suggestion
	f.StringVarP(&i.imagePullPolicy, "kudo-image-pull-policy", "", "", "Override KUDO controller image pull policy")
```

I would perfer the longer name, in case we're going to run a second image at some point (recovery controller anyone?). 

But i'd be ok with this as well.